velox/exec/tests/SparkAggregationFuzzerTest.cpp

```diff
@@ -49,7 +49,8 @@ int main(int argc, char** argv) {
   // TODO: List of the functions that at some point crash or fail and need to
   // be fixed before we can enable. Constant argument of bloom_filter_agg cause
   // fuzzer test fail.
-  std::unordered_set<std::string> skipFunctions = {"bloom_filter_agg"};
+  std::unordered_set<std::string> skipFunctions = {
```

Comment:
It would be nice to create a GitHub issue to describe the problem, add a link to it next to disabled function, then prioritize fixing the issue and enabling the function.

Reply:
OK, will do. 
> I remember you removed testing of companion functions from the Fuzzer. Did this not apply to Spark Fuzzer?

Based on above discussion, is there any known issue for companion functions in the fuzzer test? Maybe I can refer to that.

Reply:
@kagamiori Wei, is this something you have context about?

Reply:
Hi @rui-mo, I'm not aware of any known issue of companion functions in the fuzzer test. We'd appreciate it if you could create a Github issue for the error you come across.

Reply:
I'm seeing that we skip results verification for avg_partial in Presto's Fuzzer. Maybe we should do the same in Spark's Fuzzer.

```
          // TODO: Skip result verification of companion functions that return
          // complex types that contain floating-point fields for now, until we
          // fix
          // test utilities in QueryAssertions to tolerate floating-point
          // imprecision in complex types.
          // https://github.com/facebookincubator/velox/issues/4481
          {"avg_partial", ""},
          {"avg_merge", ""},
```

Reply:
@mbasmanova, Thank you for catching this! This list was recently moved out of AggregationFuzzerTest.cpp so I didn't saw it. Yes, they were added to the default customVerificationFunctions list in AggregationFuzzerRunner. However, since SparkAggregationFuzzerTest provides its own customVerificationFunctions, the default one was not used. We should let AggregationFuzzerRunner combine the default function list and the provided function list. 

@rui-mo, if the problem you come across was that the fuzzer complained about unmatched result of avg_partial, a quick fix to unblock this PR can be adding `{"avg_partial", ""}, {"avg_merge", ""}` to the customVerificationFunctions defined in this file (SparkAggregationFuzzerTest.cpp). Please let us know if the fuzzer still fails after this change.

Reply:
@kagamiori @mbasmanova Thank you. Updated customVerificationFunctions in SparkAggregationFuzzerTest.cpp, and looks there's already this issue https://github.com/facebookincubator/velox/issues/4481.